### ISSUE TITLE
DateTimes and JSON 

### ISSUE THREAD

**Issue by oliverjanik** [bchavez/RethinkDb.Driver#64]
I've got an object in the DB that looks like this:

```json
{
    "start":  "2016-05-24T17:40:00"
}
```

However when I load it through RethinkDb.Driver as JObject I get `"2016-05-24T17:40:00Z"` back. Notice the 'Z' at the end. All I want is my string back as it is.

I believe it's Json.Net doing its nonsense again.

**Comment by bchavez** [comment]
Hi Oliver,

I can't help without more information:

2. ReQL queries (+ associated classes).
3. The driver version you are using.
4. All relevant C# code, in context, relating to how the `JObject` got created, how it's used, and how it's retrieved:
   * Received From `Browser -> C#` then 
   * Sent From `C# -> RethinkDB`, then
   * Retrieved from `C# <- RethinkDB`
5. JSON protocol traces, in context, for each step.
6. Preferably, a unit test to reproduce the issue.

The best I can gauge what's happening is to your `DateTime` is: https://github.com/bchavez/RethinkDb.Driver/issues/39#issuecomment-215985635

> Thanks for the `DateParseHandling` tip.

> If I set it to `DateParseHandling.None` my timestamp should stay a string and that would avoid this roundtrip issue. It would prevent me from running time functions on the database though.

If you need additional help, I'm available on slack. You can contact me on slack anytime. Also, I hope we can, as much as possible, keep GitHub Issues related to source code changes, confirmed bugs, or feature requests.

Muchas Gracias,
Brian

:zap: :runner: [***"I was struck by lighting. Walkin' down the street..."***](https://www.youtube.com/watch?v=iypUpv9xelg)

**Comment by oliverjanik** [comment]
1. r.db('db').table('t').get('id')
2. 2.3.3
3. No classes, I'm pulling it via RunAtomAsync<JObject>

Aaaand I just realised this is my `JsonSerializerSettings` issue. 

For posterity, I had this in my settings:
```
            DateParseHandling = DateParseHandling.None,
            DateTimeZoneHandling = DateTimeZoneHandling.Utc,
```

Even though you say `DateParseHandling.None` JObject still loads `"2016-05-24T17:40:00"` as DateTime!!!

So I changed `DateTimeZoneHandling` to `DateTimeZoneHandling.Unspecified` which doesn't prevent Json.Net from parsing the string but at least it doesn't add the extra UTC bit.

Now let'see if it breaks in different scenario :+1:
